baybe/recommenders/naive.py

```diff
@@ -117,7 +116,7 @@ def recommend(  # noqa: D102
         # will then be attached to every discrete point when the acquisition function
         # is evaluated.
         cont_part = searchspace.continuous.samples_random(1)
-        cont_part = to_tensor(cont_part).unsqueeze(-2)
+        cont_part_tensor = cast(Tensor, to_tensor(cont_part)).unsqueeze(-2)
```

Comment:
Why is this cast necessary? Shouldn't `to_tensor` guarantee to return a Tensor?

Reply:
nope it is coded in a way that it returns either a `Tensor` or `Iterator[Tensor]` hence the mypy complaint here

since the iterator version is used in other parts I didnt want to change the utility and think the cast should be fine ere